dask_image/ndmeasure/_utils/_label.py

```diff
@@ -123,7 +123,12 @@ def _to_csr_matrix(i, j, n):
     return mat.tocsr()
 
 
-def label_adjacency_graph(labels, structure, nlabels, wrap=None):
+def set_tup_value(tup, idx, value):
+    """Return a copy of `tup` with `value` at `idx`."""
+    return tuple((elem if i == idx else value) for i, elem in enumerate(tup))
```

Comment:
Note that I changed this to `==`. I guess this comes down to how one thinks about wrapping. As it is now, setting `wrap_axes=(0)` wraps labels across the boundary of the 0th axis, whereas `!=` would wrap the array over the 0th axis and wrap labels across the boundary of the 1st axis.

Reply:
Yes, I agree. Just wanted to comment it since it differs from the suggestion of @jni.

Reply:
The suggestion from @jni was pretty abstract and untested and he well could have gotten a sign wrong somewhere 😅 Thanks @Holmgren825 and @m-albert! Sorry the next few weeks are very busy for me so I may not be able to do an in-depth review, but I'll try. Please ping me if there is a rush/you specifically want an extra pair of eyes on something.

Reply:
> To me, indicating the axes over which dask_image.ndmeasure.label should consider the input array to wrap over is most intuitive :) (which is what I think this code does)

I had been wrong earlier in the sense that the code above should be saying `!=` as jni suggested earlier. Because the idea is to only replace the element at index `idx` and leave other elements unchanged (those with `i != idx`).

> Sorry the next few weeks are very busy for me so I may not be able to do an in-depth review, but I'll try. Please ping me if there is a rush/you specifically want an extra pair of eyes on something.

Thanks for your availability @jni :)

Reply:
Yes, same here. I guess I got confused thinking about array axes and geographical axes for some reason. Fixed in https://github.com/dask/dask-image/pull/344/commits/3aaeaca8cad1c3523346e82b7f6440a24f5e1388.